config/routes.rb

```diff
@@ -20,7 +20,7 @@
   end
   devise_for :admins
   root 'home#index'
-  get '/collection', to: 'catalog#index', as: :org_collection
+  get '/collection', to: 'catalog#index', as: :org_collection, constraints: RestrictJsonRequest.new
```

Comment:
Any specific reason why we are applying a constraint on this route?

Reply:
Because this is only allowed with Content-Type `JSON`. It will throw an error with other content-types

Reply:
Okay

Reply:
Do we also need to check for format? `request.format == :json` or is that not required?